docs/UserGuide.md

```diff
@@ -179,14 +179,21 @@ Format: `addt INDEX d/DESCRIPTION amt/AMOUNT o/OTHER_PARTY dt/DATE`
 * Adds a transaction to the client at the specified `INDEX`
 * The index refers to the index number shown in the displayed client list.
 * The index **must be a positive integer** 1, 2, 3, …​
+* The amount should be a number of up to two decimal places containing only digits (`0 - 9`) and one decimal point (`.`) if needed.
+e.g. `10, 100.5, 1000.55`
+* If the amount includes a decimal point, there should be at least one digit before the decimal point. 
+e.g. `0.5`
+* The date should be in the format `YYYY-MM-DD` e.g. `2024-12-20`
 
 Examples:
-* `addt 1 d/buy raw materials amt/-100 o/Company ABC dt/16-10-2024`
-* `list` followed by `addt 1 d/buy raw materials amt/-100 o/Company ABC dt/16-10-2024` 
+* `addt 1 d/buy raw materials amt/-100.55 o/Company ABC dt/2024-10-16`
+* `list` followed by `addt 1 d/buy new equipment amt/-10000.5 o/ABC Motor Group dt/2024-11-17` 
 adds the transaction to the 1st client in the application.
-* `find Betsy` followed by `addt 1 d/sell raw materials amt/200 o/Company XYZ dt/01-09-2024` 
+* `find David` followed by `addt 1 d/sell raw materials amt/200 o/Company XYZ dt/2024-09-01` 
 adds the transaction to the 1st client in the results of the `find` command.
 
+![result for 'find David' followed by addt](images/addtExample.png)
```

Comment:
actually I dont think this screen shot is necessary? I think they only want us to add necessary screenshots into the UG. This is the same view as list so idt its necessary

Reply:
Ok sure then I will remove it. I am not sure if it is just on my end but the images under `list` and `findt` are not showing.
![image](https://github.com/user-attachments/assets/0c25b2bd-b928-40e3-a14d-11d8c5fb3d06)